os/include/wchar.h

```diff
@@ -66,6 +66,7 @@
 
 #include <stdio.h>
 #include <stddef.h>
+#include <sys/types.h>
```

Comment:
Could you let me know the reason of adding this header? This file does not use any definition which is defined in <sys/types.h>.

Reply:
stddef.h is missing some definition there is a comment telling that some types are in sys/types.h

I could have include sys/types.h but I dont want to pollute too much.